app/views/petitions/_content.html.erb

```diff
@@ -0,0 +1,36 @@
+<% if petition.english_translated? && petition.gaelic_translated? && gaelic_disabled? %>
```

Comment:
Can't this just be `gaelic_disabled? && petition.gaelic_translated?` - if the first condition is true then `petition.english_translated?` will also be true